meta-client/src/main/java/org/apache/helix/metaclient/constants/MetaClientKeeperException.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.helix.metaclient.constants;
```

Comment:
you should have a separate directory for exception. 

eg:
ls <HELIX-home>/metadata-store-directory-common/src/main/java/org/apache/helix/msdcommon/exception
InvalidRoutingDataException.java

Reply:
As discussed asynchronously, I will create a separate PR with these changes after these are merged